packages/signalwire/config/jest/setupTests.ts

```diff
@@ -0,0 +1,22 @@
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+jest.mock('@signalwire/core', () => {
```

Comment:
can we get rid of this mock?, not sure if it is still needed

Reply:
Removed!